docs/examples/full_chain_odd.md

```diff
@@ -1 +1,123 @@
-# ODD full chain
+# Walkthrough of the OpenDataDetector full chain example
+
+The OpenDataDatector (ODD) is fictitious silicon detector which resides in its own repository on [GitLab](https://gitlab.cern.ch/acts/OpenDataDetector). It is used for testing and as a performance baseline in Acts.
+
+Our full chain ODD example is written in Python and can be found [here](https://github.com/acts-project/acts/blob/main/Examples/Scripts/Python/full_chain_odd.py).
+
+The first step is to load the ODD detector description and to construct the detector. `getOpenDataDetectorDirectory` gives us the ODD folder within the `thirdparty` directory in Acts. We load our prefered material map and provide it to the detector construction `getOpenDataDetector`.
+
+```python
+oddDir = getOpenDataDetectorDirectory()
+
+oddMaterialMap = oddDir / "data/odd-material-maps.root"
+oddMaterialDeco = acts.IMaterialDecorator.fromFile(oddMaterialMap)
+
+detector, trackingGeometry, decorators = getOpenDataDetector(
+    getOpenDataDetectorDirectory(),
+    mdecorator=oddMaterialDeco,
+)
+```
+
+In our simple example we assume a homogenous magnetic field along the beam axis with 2 T. The magnetic field is passed to all the different algorithms in our simulation and the reconstruction pipeline.
+
+```python
+field = acts.ConstantBField(acts.Vector3(0.0, 0.0, 2.0 * u.T))
+```
+
+The simulation step involves random processes and in order to get the same results across different executions we provide our prefered random number generator with an arbitrary seed.
+
+```python
+rnd = acts.examples.RandomNumbers(seed=42)
+```
+
+All simulation and reconstruction pipelines in Acts begin with a `Sequencer`. It controls the execution of the different algorithms in the chain. We provide the number of events, the threads we want to utilize (-1 means all) and the desired log level.
```

Comment:
We should be a bit more precise as to what `numThreads=-1` means, eg. something like this:

```suggestion
All simulation and reconstruction pipelines in Acts begin with a `Sequencer`. It controls the execution of the different algorithms in the chain. We provide the number of events, the number of threads to use (`-1` to use all the machine's cores) and the desired log level.
```
I don't know if it really means use all the cores, but hopefully someone else will correct.